Source/WebKit/UIProcess/API/Cocoa/_WKWebExtension.h

```diff
@@ -36,21 +36,13 @@ WK_EXTERN NSErrorDomain const _WKWebExtensionErrorDomain WK_API_AVAILABLE(macos(
 
 typedef NS_ERROR_ENUM(_WKWebExtensionErrorDomain, _WKWebExtensionError) {
     _WKWebExtensionErrorUnknown = 1,
-    _WKWebExtensionErrorManifestNotFound,
+    _WKWebExtensionErrorResourceNotFound,
+    _WKWebExtensionErrorInvalidResourceCodeSignature,
     _WKWebExtensionErrorInvalidManifest,
     _WKWebExtensionErrorUnsupportedManifestVersion,
-    _WKWebExtensionErrorInvalidActionIcon,
-    _WKWebExtensionErrorInvalidBackgroundContent,
+    _WKWebExtensionErrorInvalidManifestEntry,
+    _WKWebExtensionErrorInvalidDeclarativeNetRequestEntry,
     _WKWebExtensionErrorInvalidBackgroundPersistence,
-    _WKWebExtensionErrorInvalidContentScripts,
-    _WKWebExtensionErrorInvalidDeclarativeNetRequest,
-    _WKWebExtensionErrorInvalidDescription,
-    _WKWebExtensionErrorInvalidExternallyConnectable,
-    _WKWebExtensionErrorInvalidIcon,
-    _WKWebExtensionErrorInvalidName,
-    _WKWebExtensionErrorInvalidURLOverrides,
-    _WKWebExtensionErrorInvalidVersion,
-    _WKWebExtensionErrorInvalidWebAccessibleResources,
```

Comment:
It looks like we still kept around some localizable strings. Were these all made one error domain/code?